Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,32 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            int x = 2;
+            while (x > 0)
+            {
+                //Thank you for reviewing my code. You guys are amazing.
+
+                Console.Write("Hello. What is your name? ");
+                string name = Console.ReadLine();
```

Comment:
Prefer to use var

Reply:
I kind of disagree. This is a bit opinion based IMO. However it is clearly possible here.